cellprofiler/modules/enhanceorsuppressfeatures.py

```diff
@@ -51,171 +54,262 @@ def create_settings(self):
+*(Used only if "{E_DARK_HOLES}" is selected)*
+
+The range of hole sizes to be enhanced. The algorithm will identify only
+holes whose diameters fall between these two values.
+""".format(**{
+                "E_DARK_HOLES": E_DARK_HOLES
+            })
+        )
 
         self.smoothing = cellprofiler.setting.Float(
-                'Smoothing scale', value=2.0, minval=0, doc="""
-            <i>(Used only for the %(E_TEXTURE)s, %(E_DIC)s or %(E_NEURITES)s methods)</i><br>
-            <ul>
-            <li><i>%(E_TEXTURE)s</i>: This is the scale of the texture features, roughly
-            in pixels. The algorithm uses the smoothing value entered as
-            the sigma of the Gaussian used to weight nearby pixels by distance
-            in the variance calculation.</li>
-            <li><i>%(E_DIC)s:</i> Specifies the amount of smoothing of the image in the direction parallel to the
-            shear axis of the image. The line integration method will leave
-            streaks in the image without smoothing as it encounters noisy
-            pixels during the course of the integration. The smoothing takes
-            contributions from nearby pixels which decreases the noise but
-            smooths the resulting image. </li>
-            <li><i>%(E_DIC)s:</i> Increase the smoothing to
-            eliminate streakiness and decrease the smoothing to sharpen
-            the image.</li>
-            <li><i>%(E_NEURITES)s:</i> The <i>%(N_TUBENESS)s</i> option uses this scale
-            as the sigma of the Gaussian used to smooth the image prior to
-            gradient detection.</li>
-            </ul>
-            <img src="memory:%(PROTIP_AVOID_ICON)s">&nbsp;
-            Smoothing can be turned off by entering a value of zero, but this
-            is not recommended.""" % globals())
+            'Smoothing scale',
+            value=2.0,
+            minval=0,
+            doc="""\
+*(Used only for the "{E_TEXTURE}", "{E_DIC}" or "{E_NEURITES}" methods)*
+
+-  *{E_TEXTURE}*: This is the scale of the texture features, roughly in
+   pixels. The algorithm uses the smoothing value entered as the sigma
+   of the Gaussian used to weight nearby pixels by distance in the
+   variance calculation.
+-  *{E_DIC}:* Specifies the amount of smoothing of the image in the
+   direction parallel to the shear axis of the image. The line
+   integration method will leave streaks in the image without smoothing
+   as it encounters noisy pixels during the course of the integration.
+   The smoothing takes contributions from nearby pixels which decreases
+   the noise but smooths the resulting image.
+-  *{E_DIC}:* Increase the smoothing to eliminate streakiness and
+   decrease the smoothing to sharpen the image.
+-  *{E_NEURITES}:* The *{N_TUBENESS}* option uses this scale as the
+   sigma of the Gaussian used to smooth the image prior to gradient
+   detection.
+
+|image0| Smoothing can be turned off by entering a value of zero, but
```

Comment:
This image is not rendering properly in my GUI

Reply:
How odd! It renders correctly in mine. I made a slight change, maybe it works now?